projects/convenience.md

```diff
@@ -0,0 +1,94 @@
+# Developer Experience Project
```

Comment:
The file is still named `convenience.md`. Should it be renamed, maybe to `developer-experience.md`?

Reply:
yup. will do.